app/views/declaration/declarationitems/items_summary.scala.html

```diff
@@ -145,7 +146,9 @@
             )),
             html = HtmlFormat.fill(immutable.Seq(
                 itemsTable,
-                addAnotherQuestion("declaration.itemsSummary.addAnotherItem.question", form),
+                if (!request.isType(CLEARANCE))
+                addAnotherQuestion("declaration.itemsSummary.addAnotherItem.question", form)
+                else Html(""),
```

Comment:
Not sure, but maybe this could be `else HtmlFormat.empty`, which seems maybe more on spot. Not a big deal btw.

Reply:
Resolved as advised.